zeebe/protocol/src/main/java/io/camunda/zeebe/protocol/record/value/TenantRecordValue.java

```diff
@@ -35,4 +35,7 @@ public interface TenantRecordValue extends RecordValue {
 
   /** Key of the entity associated with this tenant. */
   long getEntityKey();
+
+  /** The type of the entity to assign/remove from a role. */
```

Comment:
🔧 Shouldn't this be "tenant" instead of "role"?

```suggestion
  /** The type of the entity to assign/remove from a tenant. */
```